src/js/components/text-message.jsx

```diff
@@ -32,12 +34,10 @@ export class TextMessage extends Component {
                    </span>;
         });
 
-        if (this.props.text.trim() && this.props.actions.length > 0) {
-            text = <span className='has-actions'>{ text }</span>;
+        if (this.props.hasImage || this.props.hasActions) {
```

Comment:
my other comment refers to this, we use a `has-actions` class for images too, makes little sense.. we coul just pass the style we want and forget about images and actions here.